inc/saimacsec.h

```diff
@@ -692,7 +692,7 @@ typedef enum _sai_macsec_sa_attr_t
 
     /**
      * @brief MACsec Salt used for encryption/decryption.
-     * Network Byte order.
+     * Network Byte order. Used only if SAI_MACSEC_SC_ATTR_MACSEC_XPN64_ENABLE == true.
```

Comment:
make this as condition ?

Reply:
This is a MACSEC_SA attribute conditioned on a MACSEC_SC attribute.  Sanity-check did not allow that.

Reply:
aa ok, cross object conditions are not allowed